pkg/api/addons.go

```diff
@@ -243,16 +251,19 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 
 	defaultAzureNetworkPolicyAddonsConfig := KubernetesAddon{
 		Name:    AzureNetworkPolicyAddonName,
-		Enabled: azureNetworkPolicyAddonEnabled(o),
+		Enabled: to.BoolPtr(o.KubernetesConfig.NetworkPlugin == NetworkPluginAzure && o.KubernetesConfig.NetworkPolicy == NetworkPolicyAzure),
 		Containers: []KubernetesContainerSpec{
 			{
-				Name: AzureNetworkPolicyAddonName,
+				Name:  AzureNetworkPolicyAddonName,
+				Image: "mcr.microsoft.com/containernetworking/azure-npm:v1.0.18",
 			},
 		},
 	}
 
 	defaultDNSAutoScalerAddonsConfig := KubernetesAddon{
-		Name:    DNSAutoscalerAddonName,
+		Name: DNSAutoscalerAddonName,
+		// TODO enable this when it has been smoke tested
```

Comment:
Moved this // TODO comment here, as this is where it will be enabled by default in the future